package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/sarbbottam/js-ds",
   "devDependencies": {
     "chai": "^3.5.0",
+    "chai-sorted": "^0.1.0",
```

Comment:
I guess this change is related to `sort` refactor.

Please take this out of this PR and raise a separate one. [Address only one use case in one PR.](https://github.com/sarbbottam/js-ds/blob/master/contributing.md#pull-request)